### PR TITLE
Invalidate resdk.tables cache on AnnotationValue updates

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Review by marcellevstek — APPROVED** [review]
QA done on the PR. Works as expected.

File: src/resdk/resources/annotations.py [hunk 1]
```diff
@@ -92,7 +94,7 @@ class AnnotationValue(BaseResource):
 
     endpoint = "annotation_value"
 
-    READ_ONLY_FIELDS = BaseResource.READ_ONLY_FIELDS + ("label", "modified")
+    READ_ONLY_FIELDS = BaseResource.READ_ONLY_FIELDS + ("label",)
```
Comment (gregorjerse) [inline src/resdk/resources/annotations.py:97]: Why was the field `modified` removed?

File: src/resdk/resources/annotations.py [hunk 2]
```diff
@@ -115,6 +117,12 @@ def __init__(self, resolwe: "Resolwe", **model_data):
         self._sample: Optional[Sample] = None
         super().__init__(resolwe, **model_data)
 
+    @property
+    @assert_object_exists
+    def modified(self):
+        """Modification time."""
+        return parse_resolwe_datetime(self._original_values["modified"])
```
Comment (gregorjerse) [inline src/resdk/resources/annotations.py:124]: Just a remark: I believe Resolwe uses ISO 8601 datetime. So you could just use `datetime.fromisoformat(date_string)` to convert from it to the `datetime` object.

**Review by gregorjerse — APPROVED** [review]

File: src/resdk/resources/annotations.py (hunk 2 above)
Comment (JureZmrzlikar) [reply src/resdk/resources/annotations.py:124]: OK, will check this solve in one of the subsequent PRs.

**Pull request merged**